src/app/src/main/kotlin/org/vaccineimpact/api/app/controllers/PasswordController.kt

```diff
@@ -33,6 +33,7 @@ open class PasswordController(
         return setPasswordForUser(context, repo, context.username!!)
     }
 
+    @Throws(Exception::class)
```

Comment:
Why?

Reply:
So this is a little annoying - mockito won't let you setup a method on a mock to `doThrow` unless that method explicitly does throw an exception (of the required type if you `doThrow` a specific type.) Can you think of a better way of handling this?

Reply:
We could create a dummy controller with associated onetime action just for testing?

Reply:
No, I think this is fine. Maybe just add a comment to explain why it's there?